### DISCUSSION TITLE
2023/11/24/zotero-obsidian-interaction/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#5] [General]
# 2023/11/24/zotero-obsidian-interaction/

zotero是一款著名的文献管理软件，而obsidian是一个界面很好看的markdown笔记软件。这两款软件都有着丰富的插件生态，并且二者可以联动，完成许多意想不到的事情。这里记录的就是我在两款软件联动上的一点点探索。

https://wz.anoms.top/2023/11/24/zotero-obsidian-interaction/

**Comment by Hazelandinstant** [comment]
Goooood!
可惜我到现在还没搞好！
行动力低下hhhh

**Reply by cyclinbox** [reply]
如果这两个软件组合平时用不到的话，不一定要强求自己搞好呀
不过，最近有遇到些啥其他的有趣的软件，也分享分享呗hhh

**Reply by Hazelandinstant** [reply]
嘿嘿，有个新方向：希望obsidian+flomo也可以搞个联动
因为flomo的笔记导出是html格式，我想着能不能导出到obsidian的一个库里，然后根据关键词建立不同的note

**Reply by cyclinbox** [reply]
可以看看嘿嘿
下周更新的素材+1
